rephrase help page (#554)
hacs/documentation: source/docs/help/index.md

## Browse the documentation

Read the docs on the [using HACS](/docs/use/index.md), on [publishing your own custom element to HACS](/docs/publish/index.md), or on [developing HACS](/docs/contribute/index.md).

## FAQ

Maybe your question has already been answered in the [Frequently Asked Questions](/docs/faq/index.md).

## Stay up to date

Find out what's new with this project over at [GitHub](https://github.com/hacs).

## Want to submit a bug?

> If you're having an issue with a custom element, this is not the page for you. Go to the issue tracker of that repository.

Author: ```suggestion
If you're having an issue with a custom element, this is not the page for you. [Report an issue for that repository](/docs/use/repositories/dashboard.md/#reporting-an-issue-with-a-repository).
```

> If you're having an issue with a custom element, this is not the page for you. [Report an issue for that repository](/docs/use/repositories/dashboard.md/#reporting-an-issue-with-a-repository).

Author: ```suggestion
If you're having an issue with a custom element, [report an issue for that repository](/docs/use/repositories/dashboard.md/#reporting-an-issue-with-a-repository).
```
